scripts/cc-cli/src/commands/unbond.ts

```diff
@@ -35,7 +35,7 @@ async function unbondAction(options: OptionValues) {
   requireStatus(stashStatus, "bonded");
 
   // Unbond transaction
-  const tx = api.tx.staking.unbond(toMicrounits(options.amount).toString());
+  const tx = api.tx.staking.unbond(parseCTCString(options.amount).toString());
 
   const hash = await tx.signAndSend(controllerKeyring);
```

Comment:
:heavy_check_mark:  Calling `unbond` with the floating point amount I see in Polkadot seems to work

```
$ node dist/index.js unbond -a 1023903.129857933975021235  -u ws://127.0.0.1:9946 
2023-06-22 23:57:18        API/INIT: RPC methods not decorated: creditcoin_hashrate, task_getOffchainNonceKey
2023-06-22 23:57:18        API/INIT: creditcoin-node/222: Not decorating unknown runtime apis: TaskApi/1
✔ Specify a seed phrase for the Controller account … ****************************************************************************
Unbond transaction hash: 0x7e118d085159739ceab274d5ae5dc93197ec9eca9e15c621c93ff3c1a4286667
```

![image](https://github.com/gluwa/creditcoin/assets/1002300/a27bc5bd-0c9f-4b93-b304-7545117bbc4c)